website/content/en/docs/deployment/vanilla/guide-terraform.md

```diff
@@ -39,6 +52,18 @@ cluster_region="${CLUSTER_REGION}"
 EOF
 ```
 
+### Full Configuration
```

Comment:
nit: additional configurations or All configurations instead of full? Not my forte but full doesnt sound right

A full list of inputs for the terraform stack can be found <give link to the variables.tf> https://github.com/awslabs/kubeflow-manifests/blob/main/deployments/vanilla/terraform/variables.tf


Same comment for others

Reply:
ok

Reply:
done